index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Me</title>
+  <link rel="stylesheet" type="text/css" href="./assets/css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
+</head>
+
+<body>
+  <div id="grid">
+    <div class="nav">
+      <div class="name">Ben Oh</div>
+        <ul class="links">
+          <li><a href="index.html" target="_blank">About</a></li>
```

Comment:
Since your profile is only a few pages, I would recommend to take out the 'target' attribute since you don't want the user to have a billion pages. Not super important, just take UX into consideration.